benchmarks/viscoelastic_flexure_finite-length_elastic_plate/viscoelastic_flexure_finite-length_elastic_plate.prm

```diff
@@ -0,0 +1,180 @@
+# This benchmark examines flexure of a finite-length elastic plate based
+# on the setup described in Choi et al., 2013, JGR, v.108, p.2429-2444.
+# The model setup has a 50 km long and 5 km thick elastic plate 
+# (shear modulus: 30 GPa, viscosity: 1e35 Pa s) overlaying a viscous 
+# layer (viscosity: 1e17 Pa s, 12.5 km thick), which both have a density
+# of 2700 kg/m^3. The top of the model is a free surface, while the bottom
+# boundary is zero-slip and the sides are free-slip. The upper right corner 
+# of the viscous layer is replaced with a low density (1890 kg/m^3) elastic 
+# block, which drives flexure of the elastic plate (right side goes up, left 
+# side goes down). The total topography obtained after the model has reached 
+# equilibrium is 330 m, with -146.5 m and 183.5 m of topography, respectively, 
+# on the left and right sides. The summed value and profiles differ slightly 
+# than those reported in Choi et al. (see Figure 3), but closely match more 
+# recent experiments performed by Cedric Thieulot:
+# https://github.com/naliboff/fieldstone/blob/master/python_codes/fieldstone_64/results/flexureplate/topo.pdf  
+  
+# Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 1e3
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set CFL number                             = 0.1
+set Maximum time step                      = 5
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+# Solver settings
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Linear solver tolerance = 1e-7
```

Comment:
This is the default value. If there is no value in having this section I would propose to get rid of the whole subsection.